### ISSUE TITLE
Add single logout to saml apps

### ISSUE THREAD

**Issue by sme123999** [okta/terraform-provider-okta#146]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Can you add the Enable single logout and single logout urls to the okta_app_saml resource

### New or Affected Resource(s)

* okta_app_saml

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by Ashley-Gunn** [comment]
Commenting to hopefully keep this open. I see it's tagged with needs investigation, but am not seeing any movement. My team really needs this piece to fully manage our SAML apps via Terraform.

**Comment by Ashley-Gunn** [comment]
Thank you for reopening this @bogdanprodan-okta 👍

**Comment by bogdanprodan-okta** [comment]
I will try to add this field into the official Okta Golang SDK ASAP, so I can add the corresponding code here.

**Comment by bogdanprodan-okta** [comment]
I'm waiting for the changes to the specs to be merged, so this will definitely will go into v3.8.0

**Comment by bogdanprodan-okta** [comment]
Hi @Ashley-Gunn! I've created PR for this issue but still waiting for the missing field to be added to the official specs.
